packages/db/prisma/schema.prisma

```diff
@@ -74,15 +74,26 @@ model BlobStoragesState {
   @@map("blob_storages_state")
 }
 
-model Address {
-  address                    String   @id
-  firstBlockNumberAsSender   Int?     @map("first_block_number_as_sender")
+model AddressHistory {
+  address                    String
+  category                   Category @default(OTHER)
```

Comment:
Category is defined as nullable in other models so I think it would be more coherent to do it here too